hardware_interface/src/resource_manager.cpp

```diff
@@ -287,6 +287,7 @@ class ResourceStorage
 
     if (result)
     {
+      remove_all_hardware_interfaces_from_available_list(hardware.get_name());
       async_component_threads_.erase(hardware.get_name());
       // TODO(destogl): change this - deimport all things if there is there are interfaces there
       // deimport_non_movement_command_interfaces(hardware);
```

Comment:
```suggestion
```
could all these comments be removed?

Reply:
No, not yet. We don't do any deimporting now. This should happen when we start implementing support for configuration interfaces so that the “movement” interfaces are only available when hardware is active.

Reply:
ok!